### PR TITLE
Add blog post "0.1 + 0.2 is not 0.3..."

### LINKED ISSUE #47: [BLOG POST] 0.1 + 0.2 is not equal to 0.3
# Title: 0.1 + 0.2 and other fantastic ways to make money disappear
# Abstract:
Type `0.1 + 0.2 == 0.3` into the Python interpreter. Yep. It's false. Before you throw away Python, try it out in C. And C++. Also Java, Go, Rust, Julia or Assembly.

Let's explore why this bug in all languages that can never be "fixed", and how we go around it to make sure stock markets don't collapse and your weapons of mass destruction don't hit the wrong place.

## Who is your target audience with this post? Be as specific as you can.
All programmers who have to concern themselves with floating point calculations. This is some important stuff that nobody taught us which is a big sad.

### PR DESCRIPTION
**Pull request by tfidfwastaken**
Closes #47 

I realized that given the length and formatting of my blog post, it would be really tricky to review it, so I have provided a slightly easier to read version [here](https://loosi.herokuapp.com/l/vzDkPUKGwvz0). The images won't load on it though, because they are located within the repo, so unfortunately you'll have to view the images separately by accessing `/assets/images/floats/` on the github source browser.

Please give feedback when you can, thanks!

Assigning @Gituser143 for review, and cc @anihm136

### CODE REVIEW

**Comment by Gituser143** [comment]
@tfidfwastaken I absolutely love the post! No changes I could think of though. Maybe have it reviewed by another set of eyes? In case I missed some spelling mistake or something xD

**Comment by tfidfwastaken** [comment]
Yeah I already found a bunch of spelling mistakes, and fixed them in that new commit.

@siddhantrao23, can you be the other set of eyes?

**Comment by anihm136** [comment]
This is super unique and very readable despite the length! Just for information, the video I mentioned had another interesting solution to the problem by representing a float as a pair of integers - one representing the magnitude and the other a power of 10 to multiply.

**Comment by tfidfwastaken** [comment]
@anihm136 Yeah, given the length at which the post was going I decided to leave details like those out in my "further reading" section. The first link also discusses the drawbacks and advantages for the integer trick you mentioned.

My main objective was to just signal attention to the fact that floating point is far more nuanced than we are often made to believe.

File: _posts/2020-02-22-01-plus-02-is-not-03.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+
+“Why are you using a float?”
+
+“I don’t know, because that’s what the game does?”
+
+“That’s the problem. I read about this in a blog the other day.”
+
+“Bu-but there’s no problem in *this* code, it prints 100000.”
+
+“Ay you wait da,” she said while ripping a page from her lab observation book.
```
Comment (siddhantrao23) [inline _posts/2020-02-22-01-plus-02-is-not-03.md:167]: Was the "," intentional?

Overall, I really appreciate this post! Nicely done @tfidfwastaken

Comment (tfidfwastaken) [reply _posts/2020-02-22-01-plus-02-is-not-03.md:167]: Yep, that's how it's done usually.

Even I was surprised by that, I opened a book lying around, the convention is comma inside quotation.

**Comment by tfidfwastaken** [comment]
Okay it looks like you all are okay with the merge 

Thanks!

**Pull request merged**